### DISCUSSION TITLE
CoPilot having a hard time suggesting translation in `.yaml`

### DISCUSSION THREAD

**Discussion by irfandyj** [community/community#46858] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

So I tried using `.yaml` file in my translation for easy access. But when I tried doing so, apparently CoPilot did not suggest anything. I tried pressing `ALT + \`, no suggestion. But when I try `ALT + ENTER`, it does have some suggestion **but,** it's a copy of other code instead of solving what I want.

I tried doing the same thing in `.js` file and it works fine.

**Comment by DurandSacha** [comment]
Faut il activer un paramétrage dans l'extension copilot pour obtenir de bons .yaml ?

**Comment by iamcjwilliams** [comment]
@jippy89 Thank you for your feedback, I will notify our copilot team
